dataactcore/interfaces/function_bag.py

```diff
@@ -649,6 +649,26 @@ def get_last_validated_date(submission_id):
     return oldest_date.strftime('%m/%d/%Y') if oldest_date else oldest_date
 
 
+def get_fabs_meta(submission_id):
+
+    sess = GlobalDB.db().session
+
+    total_rows = sess.query(DetachedAwardFinancialAssistance).filter(
+                        DetachedAwardFinancialAssistance.submission_id == submission_id)
+
+    valid_rows = total_rows.filter(DetachedAwardFinancialAssistance.is_valid)
+
+    publish_date = sess.query(CertifyHistory).filter(CertifyHistory.submission_id == submission_id).first()
```

Comment:
Because someone (probably me) started typing "last" switched to "latest" have way and no one noticed the spelling error.